backend/gce.go

```diff
@@ -109,6 +109,7 @@ var (
 cat > ~travis/.ssh/authorized_keys <<EOF
 {{ .SSHPubKey }}
 EOF
+sudo chown -R travis:travis ~travis/.ssh/
```

Comment:
Given that this script is _supposedly_ executed by `root`, I think I would prefer a version that omits `sudo` so that we aren't baking in a requirement for `sudo` to be present.

Reply:
Thanks for looking at this! We will test to make sure that it consistently succeeds without `sudo` and update the PR accordingly.